### ISSUE TITLE
File manager not working with Fancybox 4 

### ISSUE THREAD

**Issue by andryshok** [trippo/ResponsiveFilemanager#724]
Hello. File manager not working with Fancybox 4  ....   and not work callback function for select file
Any ideas?

**Comment by jshster** [comment]
Andry, have you checked your Javascript console for errors and your PHP error log?   Also what version of RFM are you using?

**Comment by andryshok** [comment]
There are no errors in the console. But as I understand it, this is due to the fact that fancybox 4 not  uses jquery. What could be the way out of this?

**Comment by jshster** [comment]
What version of the file manager are you using?  I'm going to guess you're on Version 9.14 but there have been a heap of recent updates that for whatever reason haven't even triggered an update to the minor version number.   Hard to say which one.   The other thing you didn't mention was what's actually going wrong.  You say "doesn't work".  What specifically is going wrong?  Please be specific with steps to replicate.

**Comment by andryshok** [comment]
Now, fancybox not use jquery [https://jsfiddle.net/tgcpey0m/](url)
Filemanager close window function 
`parent.jQuery.fancybox.getInstance().close()`
but this code not working for actual fancybox version  [https://fancyapps.com/fancybox/](url).
I use file manager vesion - master from this repo.
It would be good if the project was updated at least somehow, but this does not happen, I could not find alternatives open source and work with php 8^. The author abandoned the project, and I do not have enough knowledge to make big edits there.

**Comment by jshster** [comment]
OK... yep, I think I've run into the same issue.   I've started using an alternative to Fancybox which means I had to code my own close method.   Don't think the project has been abandoned, as Alberto has been making commits in the past couple of weeks.   Just nothing official so don't lose hope! I'm the same as you.  I haven't found an alternative yet that has anywhere this level of functionality or doesn't cost the earth or have pricing structures that don't work for a freelancer!   

Leave it with me and I'll see if I can find the answer.

**Comment by andryshok** [comment]
I agree with you. It's good if the project lives. I can't give up fancybox4, but I think to come up with workarounds with the wrong solution, so I think how to turn to fancybox4 without jquery.
